### DISCUSSION TITLE
Bed Leveling

### DISCUSSION THREAD

**Discussion by Kryosyss** [Jyers/Marlin#1227] [Q&A] [answered]
Hi everyone ! :)
First of all I apologize for my English simply translated by google trad, I am French.
Then congratulations for this great firmware that I have been using for a few weeks, it's perfect.

I have an ender3 v2 with a 4.2.7 motherboard, so I flashed the pre-compiled firmware "E3V2-BLTouch-3x3-v4.2.7-v1.3.4.bin"
Before that I had version 1.3.3
I have a question or rather something that I do not understand concerning the bed Leveling with the Bltouch, I set the Z-Offset via the menu provided for this purpose, I save.
Then I have a new mesh created automatically with the menu, everything is fine.

However when I run a print (g29 included in the gcode or not) my nozzle is still too high but really not very much, maybe 0.2 / 0.5 mm, why is that?
have any idea where it might come from? I am using Cura or PrusaSlicer, everything was fine, I have no Z offset set in the slicer.

Thanks for your help, I'm sure it's a silly thing, but I can't quite put my finger on it.

**Comment by Jyers** [comment, marked as answer]
Could be from inconsistency when setting Z offset, or some issue upstream. Maybe it's fixed with the latest source :)

**Reply by Kryosyss** [reply]
Hi, I took some time for 3d printing, I performed the last firmware update and the issue seems to have been fixed.
however I am amazed to see that after updating the firmware and resetting the configuration, the print count statistics are not reset, how do I reset the statistics?

Thank you for the work done, it's excellent.

**Reply by Jyers** [reply]
Haha, yes they are meant to be persistent but I do need to add an option for resetting. Will be coming soon! Super glad the issue is fixed :)
